Poker.BE/Poker.BE.Domain/Game/Room.cs

```diff
@@ -22,7 +22,6 @@ public class Room
 
         #region Fields
         private ICollection<Player> activeAndPassivePlayers;
-        private Deck deck;
```

Comment:
the Hand creates the deck.. just thought it makes more sense
anyway this doesn't change anything